R/utils.R

```diff
@@ -118,3 +118,23 @@ benchmarks <- function() {
     "unit_tilt_sector"
   )
 }
+
+
+#' Create a dictionary for brittle columns names
+#'
+#' @return A data frame.
+#' @noRd
+dictionary <- function() {
+  tribble(
+    ~aka, ~column,
+    "emission_profile", "emission_profile",
+    "profile_ranking", "profile_ranking",
+    "transition_risk_score", "transition_risk_score"
+  )
+}
```

Comment:
I see the allignment is off.

You may format this as a spredsheet easily with `datapasta::tribble_paste()` then protect the formatting with `# styler: off` and `# styler: on`.

``` r
# https://styler.r-lib.org/articles/styler.html#ignoring-certain-lines
# Protect format
# styler: off
# Small datasets you can format by hand.
# Large datasets are easy to format with `datapasta::tribble_paste()`
data <- tibble::tribble(
    ~x, ~y,
   "a",  1,
)
# styler: on
```

<sup>Created on 2024-04-16 with [reprex v2.1.0](https://reprex.tidyverse.org)</sup>